app.js

```diff
@@ -2,16 +2,134 @@
+
+    console.log(
+        `gameState: ${gameState}`,
+        `guess: ${guess}`,
+        `result: ${result}`,
+        `pearl: ${pearl}`
+    ); 
+    */
+
+    if (gameState === 'guess') {
+        guesses.classList.remove('hidden');
+        results.classList.add('hidden');
+        playAgainBtn.classList.add('hidden');
+    } else if (gameState === 'results') {
+        guesses.classList.add('hidden');
+        results.classList.remove('hidden');
+        playAgainBtn.classList.remove('hidden');
+        if (guess === 1) {
+            shell1.classList.add('reveal');
+            if (result === 'win') {
+                display1.textContent = 'Found it!';
+                pearl1.classList.remove('hidden');
+            } else {
+                display1.textContent = 'Not Here!';
+                if (pearl === 2) {
+                    shell2.classList.add('reveal');
+                    pearl2.classList.remove('hidden');
+                } else {
+                    shell3.classList.add('reveal');
+                    pearl3.classList.remove('hidden');
+                }
+            }
+        } else if (guess === 2) {
+            shell2.classList.add('reveal');
+            if (result === 'win') {
+                display2.textContent = 'Found it!';
+                pearl2.classList.remove('hidden');
+            } else {
+                display2.textContent = 'Not Here!';
+                if (pearl === 1) {
+                    shell1.classList.add('reveal');
+                    pearl1.classList.remove('hidden');
+                } else {
+                    shell3.classList.add('reveal');
+                    pearl3.classList.remove('hidden');
+                }
+            }
+        } else {
+            shell3.classList.add('reveal');
+            if (result === 'win') {
+                display3.textContent = 'Found it!';
+                pearl3.classList.remove('hidden');
+            } else {
+                display3.textContent = 'Not Here!';
+                if (pearl === 1) {
+                    shell1.classList.add('reveal');
+                    pearl1.classList.remove('hidden');
+                } else {
+                    shell2.classList.add('reveal');
+                    pearl2.classList.remove('hidden');
+                }
+            }
+        }
+    }
+}
+
+function liftShell(userGuess) {
+    if (userGuess === pearl) {
+        // Win
+        // Lift the shell with pearl
+        // Display 'Found It!'
+    } else {
+        // Lose
+        // Lift the shell with pearl
+        // Lift the shell guessed
+        // Display 'Not Here!'
+    }
+    gameState = 'results';
+}
+
+function loadPage() {
+    displayShells(gameState);
+}
+
+// function playAgain() {} TODO
+
 // event listeners
+guess1.addEventListener('click', () => {
+    liftShell('1');
+});
+guess2.addEventListener('click', () => {
+    liftShell('2');
+});
+guess3.addEventListener('click', () => {
+    liftShell('3');
+});
```

Comment:
part of the idea of working vertically would be adding an event listenter to guess1, and then working through whatever is happening on that event (so, finishing out the logic for ```liftShell()``` before adding eventlisteners to ```guess2``` and ```guess3```